pkg/kubelet/stats/cadvisor_stats_provider.go

```diff
@@ -303,20 +300,58 @@ func isPodManagedContainer(cinfo *cadvisorapiv2.ContainerInfo) bool {
 
 // getCadvisorPodInfoFromPodUID returns a pod cgroup information by matching the podUID with its CgroupName identifier base name
 func getCadvisorPodInfoFromPodUID(podUID types.UID, infos map[string]cadvisorapiv2.ContainerInfo) *cadvisorapiv2.ContainerInfo {
-	for key, info := range infos {
+	if info, found := infos[cm.GetPodCgroupNameSuffix(podUID)]; found {
+		return &info
+	}
+	return nil
+}
+
+// filterTerminatedContainerInfoAndAssembleByPodCgroupKey returns the specified containerInfo but with
+// the stats of the terminated containers removed and all containerInfos assembled by pod cgroup key.
```

Comment:
can we make explicit in the godoc for the two maps returns what is the key -> value mapping for each of the two maps?